DataFormats/MuonReco/src/MuonSelectors.cc

```diff
@@ -1004,6 +1022,7 @@ void muon::setCutBasedSelectorFlags(reco::Muon& muon,
   if (dbCorectedRelIso<0.20)    selectors |= reco::Muon::PFIsoMedium;
   if (dbCorectedRelIso<0.15)    selectors |= reco::Muon::PFIsoTight;
   if (dbCorectedRelIso<0.10)    selectors |= reco::Muon::PFIsoVeryTight;
+  if (dbCorectedRelIso<0.05)    selectors |= reco::Muon::PFIsoVeryVeryTight;
```

Comment:
The point is ultra-minor... However, I let you decide if you want to fix the spelling of the name of this variable ("Corected" -> "Corrected") or not. Just either fix it or let us know that you don't plan doing it.
(I have no other issues with this PR)